client/common/sass/components/_blog-card-special.sass

```diff
@@ -0,0 +1,57 @@
+.special-blog-card
+	display: grid
+	grid-template: "container"
+	overflow: hidden
+	border-top: 1px solid
+	border-color: $black
+	height: 600px
+	cursor: pointer
+
+	--scale: 1
+	--title-text-decoration: none
+
+	&:hover
+		--scale: 1.08
+		--title-text-decoration: underline
+
+	> *
+		grid-area: container
+
+	img
+		width: 100%
+		height: 100%
+		object-fit: cover
+		place-self: center
+		transition: transform 0.15s ease-in-out
+		transform: scale(var(--scale))
+
+	&__title
+		margin: 0
+		line-height: 1.2
+		font-size: $font-size-h2
+		color: $white
+		text-decoration: var(--title-text-decoration)
+		+desktop-up
+			font-size: $font-size-h3-desktop
```

Comment:
It looks to me in the Figma like the heading is larger than this and not bold weight

Reply:
I see the lack of bold, now, but when I look at the font size in Figma (by selecting the text element in the design) it says 60px (see far right column):

![image](https://github.com/freedomofpress/pressfreedomtracker.us/assets/561931/578c9f3f-6761-46c9-8405-55015f227fd0)

Which, it seems like this `$font-size-h3-desktop` gives us?

https://github.com/freedomofpress/pressfreedomtracker.us/blob/d16414ffff77eec57d74cca5b9e9f928bad62195/client/common/sass/_typography.sass#L20

That was my reasoning, anyway. Happy to change it to something else but I'm not sure what.

Reply:
probably just a visual artifact of the screenshot size then—go ahead and keep it